pkg/files/render.go

```diff
@@ -69,17 +82,7 @@ func renderFile(cfg *config.File) error {
 
 	defer func() { _ = out.Close() }()
 
-	data := templateData{
-		Env:    make(map[string]string),
-		Params: cfg.Parameters,
-	}
-
-	for _, e := range os.Environ() {
-		e := strings.SplitN(e, "=", 2)
-		if len(e) > 1 {
-			data.Env[e[0]] = e[1]
-		}
-	}
+	data := newTemplateData(cfg.Parameters)
 
 	err = tpl.Execute(out, &data)
```

Comment:
I have the suspicion that this might fail (the test coverage for this package is not... perfect 😐, and I'm unsure if the existing tests actually cover this part), as we're now passing a pointer to a pointer into `Execute` (`data` now is already a pointer type).

```suggestion
	err = tpl.Execute(out, data)
```